servers/rendering/rendering_server_globals.h

```diff
@@ -41,13 +41,13 @@ class RendererScene;
 
 class RenderingServerGlobals {
 public:
-	static RendererStorage *storage;
-	static RendererCanvasRender *canvas_render;
-	static RendererCompositor *rasterizer;
+	static RendererStorage *storage = nullptr;
+	static RendererCanvasRender *canvas_render = nullptr;
+	static RendererCompositor *rasterizer = nullptr;
```

Comment:
Static variables shouldn't be initialized in this way.
For now it is better to just remove initialization in this lines

Reply:
Yeah. I learnt this right now while going through the build error logs. I am correcting them right now. Sorry.